prototypes/docusaurus/src/css/custom.css

```
@@ -1,31 +1,157 @@
-  --ifm-code-font-size: 94%;
-  --docusaurus-highlighted-code-line-bg: rgba(8, 40, 55, 0.08);
+  --ifm-color-primary: #0969da;
+  --ifm-color-primary-dark: #085cc0;
+  --ifm-color-primary-darker: #0755b4;
+  --ifm-color-primary-darkest: #064690;
+  --ifm-color-primary-light: #1f7ae0;
+  --ifm-color-primary-lighter: #2d84e3;
+  --ifm-color-primary-lightest: #4f99e8;
+  --ifm-background-color: #ffffff;
+  --ifm-background-surface-color: #f6f8fa;
+  --ifm-font-family-base: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
```

Stylelint (17.4.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)